app/components/search/SearchResults/SearchResults.jsx

```diff
@@ -194,9 +161,13 @@ const SearchResult = ({ hit, index, setCenterCoords }) => {
           <Link to={`/organizations/${resourceId}`}>{hit.service_of}</Link>
         </div>
         <div className={styles.address}>{renderAddressMetadata(hit)}</div>
-        <ReactMarkdown className={`rendered-markdown ${styles.description}`} source={hit.long_description} />
+        <ReactMarkdown className={`rendered-markdown ${styles.description}`} source={hit.long_description} linkTarget="_blank" />
```

Comment:
Adding this linkTarget attr on our markdown editor component will instruct links to open in new tabs. This was a product request from Tammy, which I think makes a lot of sense.